DDL.sql

```diff
@@ -0,0 +1,263 @@
+    riderTwoCurrentWeight FLOAT DEFAULT NULL,
+    humidity FLOAT DEFAULT NULL,
+    driverFeedback TEXT DEFAULT NULL
+);
+
+CREATE TABLE BMSNumAttemptsMade(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSNumThermAttemptsMade(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP NOT NULL
+);
+
+CREATE TABLE BMSLastAttemptTime(
+    ID SERIAL PRIMARY KEY,
+    time INTEGER NOT NULL,
+    timeStamp TIMESTAMP NOT NULL,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSThermAttemptTime(
+    ID SERIAL PRIMARY KEY,
+    time INTEGER NOT NULL,
+    timeStamp TIMESTAMP NOT NULL,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSTotalVoltage(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSBatteryVoltage(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSCurrent(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSThermistorTemp(
+
+    ID SERIAL,
+    thermNumber INTEGER,
+    PRIMARY KEY (ID, thermNumber),
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSMinPackTemp(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER,
+    packId INTEGER,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSMaxPackTemp(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER,
+    packId INTEGER,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSBqInternalTemp(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSBqTemp(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    bqId INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSCellVoltage(
+    ID SERIAL,
+    cellId INTEGER,
+    PRIMARY KEY (ID, cellId),
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSMinCellVoltage(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    cellId INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSMaxCellVoltage(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    cellId INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSBqStatus(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
+);
+
+CREATE TABLE BMSErrorRegister(
```

Comment:
Technically, the individual bits in the error register mean different things, so we shouldn't just put it into one table. We should have a discussion about what makes the most sense here, but I think we might need more tables